docsrc/admin/installation.md

```diff
@@ -29,19 +49,25 @@ If you feel SELinux is important to the security of your server, please start a
 ```
 
 ## Create a configuration repository
-* Log back into the server after rebooting. Create a git repository to track your Data Library configuration. At IRI we call ours `dlconfig`.
+
+* Log back into the server after rebooting. Create a git repository to track your Data Library configuration. At IRI we
+  call ours `dlconfig`.
 
         mkdir dlconfig
         cd dlconfig
         git init
 
-* Inside the new git repository, install the IRIDL ansible collection:
+* Inside the new git repository, install the IRIDL ansible collection and dependencies:
 
         ansible-galaxy collection install \
             -p . \
             git+https://github.com/iridl/iridl-ansible.git
 
-* The previous command should have downloaded the collection to a subdirectory called `ansible_collections`. Add and commit that directory to your git repository, to ensure that you will use the same version of the collection every time you run the playbook.
+        ansible-galaxy collection install -p . community.docker
```

Comment:
From the docs, it looks like you can install multiple collections with the  same command.